README.md

```diff
@@ -55,3 +62,15 @@ try {
 
 ```
 
+## Limitations
+
+- Non-ember npm dependencies are not supported.  It is assumed all resolutions one of:
+   - classic ember apps
+   - classic ember addons
+   - v2 ember addons
+
+- No support for app namespace merging: so files in an addon's `app` directory will not be resolved.
+
+- No support for path conflict resolution.  This is unlikely to affect anybody, but if you have an Ember app with `my-app/app/tests/foo`, `my-app/tests/foo` will not resolve to this file, but instead only look for `my-app/tests/foo`
+
+- "main" imports are not supported (i.e. imports without paths)
```

Comment:
This is easy to add support for and we should just do it in a follow on PR

Reply:
Mind making a new issue to track that?

Reply:
#10